### ISSUE TITLE
[COGNITION]: GIBCT EYB - CONSIDER updating some abbreviations in On the job training for better understanding

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#10202]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

A few of the abbreviations on the on the job training are being read very literally by screen readers, and are not as clear as they could be. I'd like to consider updating these. Screen shots attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Screen readers announce text as natural language, not what is literally on the screen. Screenshots have the requested changes. Please disregard any that are not changed or cannot be changed.

## Environment
* https://staging.va.gov/gi-bill-comparison-tool/profile/10484047

## WCAG or Vendor Guidance (optional)

* [Abbreviations: Understanding SC 3.1.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-located.html)

## Screenshots or Trace Logs

![Screen Shot 2020-06-12 at 4 27 06 PM](https://user-images.githubusercontent.com/934879/84548291-e72bcc80-acca-11ea-9328-8f60284ded46.png)

---
![Screen Shot 2020-06-12 at 4 29 48 PM](https://user-images.githubusercontent.com/934879/84548297-e85cf980-acca-11ea-9620-cc506e5e7e64.png)

**Comment by ksantiagomorgan6** [comment]
pull request: https://github.com/department-of-veterans-affairs/vets-website/pull/13290

**Comment by delli-gatti-michael** [comment]
local testing complete

**Comment by delli-gatti-michael** [comment]
@1Copenut This is on staging and ready for review

**Comment by 1Copenut** [comment]
@delli-gatti-michael Confirmed both of these in staging, screenshots attached below. Moving to close.

---
![Screen Shot 2020-07-08 at 2 56 38 PM](https://user-images.githubusercontent.com/934879/86964318-7a6deb80-c12b-11ea-8808-75f4c2890c98.png)

---
![Screen Shot 2020-07-08 at 2 56 59 PM](https://user-images.githubusercontent.com/934879/86964325-7b9f1880-c12b-11ea-93a3-883e1d0a04e1.png)
